scss/_pagination.scss

```diff
@@ -6,9 +6,21 @@
 //
 .breadcrumb {
   background-color: $breadcrumb-bg;
+
+  &::before {
+    float: left; // Suppress inline spacings and underlining of the separator
+    padding-right: $breadcrumb-item-padding;
+    // stylelint-disable-next-line
+    font-family: "Material Icons Sharp";
```

Comment:
Now that I see the font-family, I think these icons should be optional, since material icon font is not explicitly required in arizona bootstrap.